### PR TITLE
Plot Component Guide

### PR DESCRIPTION
**Pull request by dawoodkhan82**
# Description
Guide for creating plot component. Using matplotlib, plotly, bokeh. Need to add more cool plot demos.

Please include: 
* relevant motivation
* a summary of the change 
* which issue is fixed. 
* any additional dependencies that are required for this change.

Closes: # (issue)
#1993

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: guides/5)other_tutorials/how_to_use_plot_component.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+* `fn`: the prediction function that is used when the user clicks submit. In our case this is the `outbreak` function.
+* `inputs`: a list of inputs including a dropdown menu for selecting which plotting library should be used; as well as other inputs used for plotting the output.
+* `outputs`: a plot output component.
+* `examples`: list of valid inputs from the 5 input components.
+* `cache_examples`: saves the predicted output for the examples, to save time on inference.
+
+
+## Exploring mode complex Plot Demos:
+
+TODO
```
Comment (dawoodkhan82) [inline guides/5)other_tutorials/how_to_use_plot_component.md:96]: @abidlabs @AK391 Have you seen any cool plot demos on spaces I can link here?

**Comment by abidlabs** [comment]
@dawoodkhan82 I think this Guide is a really good opportunity to walk users through how to create plots with `matplotlib` and `plotly` (I would leave out `bokeh` to avoid confusion, since we don't currently support it. We can add a note saying it will be supported in the future).

As it's currently structured, however, this Guide, only provides a single (relatively complex) example of using both together. I would avoid that and instead break this Guide into 2 sections, for `matplotlib` plots and `plotly` plots separately (since a user is probably looking for how to use one of those frameworks).

In each section, I would provide:
* Some background of the library (`matplotlib` vs. `plotly`) -- what is the advantage / disadvantage of each library
* What object should be returned for that library (e.g. a `matplotlib.pyplot.Figure` object)
* A **simple** example showing that library being used with Gradio

At the end, you could have the `plot_outbreak` demo combining both of the plotting frameworks together.

**Comment by dawoodkhan82** [comment]
> 

makes sense. Will update

**Comment by freddyaboulton** [comment]
Thanks for starting the guide @dawoodkhan82 ! I know plotting isn't strictly related to data science but what are your thoughts on creating a Data Science "track" to the guides and adding this guide to that? Would help onboard potential data scientists.

**Comment by dawoodkhan82** [comment]
@freddyaboulton Yup makes sense, I assume we'll need more guides when we do the additional integrations; like skops.

**Pull request closed without merging**